google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -35,6 +35,7 @@
 public final class StorageException extends BaseHttpServiceException {
   private static final String INTERNAL_ERROR = "internalError";
   private static final String CONNECTION_CLOSED_PREMATURELY = "connectionClosedPrematurely";
+  private static final String BATCH_RETRY_ERROR = "batchError";
```

Comment:
I think it's better to rename BATCH_RETRY_ERROR to BATCH_ERROR to follow the style.

Reply:
Done.